### PR TITLE
Groovy to KTS migration - test solution

### PR DESCRIPTION
**Pull request by android-dev-lxl**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
```
Comment (johnshea) [inline app/build.gradle.kts:44]: nit: move 2nd argument to a new line.

File: app/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+        freeCompilerArgs += "-opt-in=androidx.compose.material3.ExperimentalMaterial3Api"
```
Comment (johnshea) [inline app/build.gradle.kts:54]: Is this still needed?

File: build.gradle.kts [hunk 3]
```diff
@@ -15,14 +15,14 @@
...
+        set("compose_compiler_version", "1.4.3")
     }
 
 }
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.application' version '8.0.0' apply false
-    id 'com.android.library' version '8.0.0' apply false
-    id 'org.jetbrains.kotlin.android' version '1.8.10' apply false
+    id("com.android.application") version "8.0.1" apply false
```
Comment (johnshea) [inline build.gradle.kts:25]: Update all versions

**Review by johnshea — CHANGES_REQUESTED** [review]
`settings.gradle.kts` is missing its license.
`AndroidManifest.xml` remove blank line in `<intent-filter>` (line 35)
`themes.xml` remove blank line on line 18

File: app/build.gradle.kts (hunk 1 above)
Comment (android-dev-lxl) [reply app/build.gradle.kts:44]: I can move it, when I reformat the file AS removes this line.
There is no point in keep adding the line

**Review by johnshea — APPROVED** [review]

**Pull request merged**